### PR TITLE
Mega Merge [main <- mega_merge]

### PR DESCRIPTION
**Pull request by wheatleyinabox**

### CODE REVIEW

File: README.md
Comment (GuyWhoCode) [inline README.md:1]: Duplication of content here

File: package.json [hunk 1]
```diff
@@ -32,24 +32,33 @@
...
     "webpack-dev-server": "^4.10.1"
   },
   "dependencies": {
+    "@azure/msal-browser": "^2.38.0",
+    "@azure/msal-react": "^1.5.9",
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.14.0",
     "@mui/material": "^5.10.3",
+    "msal": "^1.4.18",
```
Comment (GuyWhoCode) [inline package.json:60]: `msal`and `msalx` aren't libraries that I used for Microsoft authentication... added on accident?

File: src/App.tsx
Comment (GuyWhoCode) [inline src/App.tsx:1]: The only files that should be in the `src` folder are `App.tsx`, `Content.tsx`, and `Index.tsx`. Try to compare your changes to dev and make sure everything's up to date.

**Review by GuyWhoCode — CHANGES_REQUESTED** [review]
Needs some changes

**Review by Leaversa — CHANGES_REQUESTED** [review]
Theres some discrepancies here between whats in dev and whats being pushed in. 
For instance ListItem and ProfessorPopup were refactored into src/components/.

I believe this original merge branch was made before a series of changes. Try to make sure that this branch is up to date to dev then start the merge process.

**Review by OJ101003 — CHANGES_REQUESTED** [review]
As others have pointed out, there seem to be some differences in file locations and in the contents of some files, an example with public/manifest.json being different. 
It looks like this branch might be outdated in some areas.

File: package.json (hunk 1 above)
Comment (wheatleyinabox) [reply package.json:60]: Huh, okay I'll remove those

File: README.md
Comment (wheatleyinabox) [reply README.md:1]: Removed that! :D

File: src/App.tsx
Comment (wheatleyinabox) [reply src/App.tsx:1]: Just updated the directory and files, and also ran `npm i` to take care of errors!

**Pull request closed without merging**